src/components/routes/homepage/ImagePicker.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button, Container, FormLabel, makeStyles } from '@material-ui/core';
+import { CloudUpload } from '@material-ui/icons';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%'
+  },
+  button: {
+    backgroundColor: theme.palette.primary.light,
+    color: '#fff',
+    width: '100%',
+    position: 'relative',
+    '&:hover': {
+      backgroundColor: theme.palette.primary.main
+    }
+  },
+  inputImage: {
+    display: 'none'
+  }
+}));
+
+export const ImagePicker = ({ imageBase64, setImageBase64 }) => {
+  const classes = useStyles();
+
+  return (
+    <Container className={classes.root}>
+      <input
+        accept="image/*"
+        className={classes.inputImage}
+        id="event-image"
+        type="file"
+        onChange={({ target: { files } }) => {
+          const imageFile = files[0];
+          if (!imageFile) return;
+          const fReader = new FileReader();
+          fReader.readAsDataURL(imageFile);
+          fReader.onloadend = ev => setImageBase64(ev.target.result);
+        }}
```

Comment:
maybe should check for multiple image files as well, in which case we reject all images and warn the user, feels like better ux

Reply:
yepp, agreed, will do

Reply:
@98sean98 yoo sean, so for this issue actually by default the `<input>` element limits the number of file which could be uploaded within one single upload (default to 1 file max). One has to put a `multiple` attribute to allow multiple files upload. See this [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file) for your reference

Reply:
great research!